### PR TITLE
remove members from org, policy, specs

### PR DESCRIPTION
**Pull request by yunixon**

### CODE REVIEW

File: spec/features/remove_member_spec.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
+require 'spec_helper'
+
+describe 'delete transaction', js: true do
```
Comment (AlexanderUlitin) [inline spec/features/remove_member_spec.rb]: @yunixon name this case properly ))

Comment (AlexanderUlitin) [inline spec/features/remove_member_spec.rb]: @yunixon also for speed-up this test, I would suggest you to add `js: true` only to contexts where you really need this. I see about half of contexts where you don't need js enabled.

File: spec/features/remove_member_spec.rb [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    it 'user can not remove yourself' do
+      expect(page).to_not have_content('Remove')
+    end
+
+    context 'user can not remove members with owner role' do
+      let!(:member_owner) { create :member, :owner, user: other_user, organization: org }
+
+      before do
+        visit members_path
```
Comment (AlexanderUlitin) [inline spec/features/remove_member_spec.rb]: @yunixon I think user already is on `members_path` here. Look at line 93.

File: spec/features/remove_member_spec.rb [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      end
+
+      it { expect(page).to_not have_content('Remove') }
+    end
+
+    context 'user can not remove members with admin role' do
+      let!(:member_admin) { create :member, :admin, user: other_user, organization: org }
+
+      before do
+        visit members_path
```
Comment (AlexanderUlitin) [inline spec/features/remove_member_spec.rb]: @yunixon I think user already is on members_path here. Look at line 93.

File: spec/features/remove_member_spec.rb [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+      end
+
+      it { expect(page).to_not have_content('Remove') }
+    end
+
+    context 'user can not remove other members with user role' do
+      let!(:other_member_user) { create :member, :user, user: other_user, organization: org }
+
+      before do
+        visit members_path
```
Comment (AlexanderUlitin) [inline spec/features/remove_member_spec.rb]: @yunixon I think user already is on members_path here. Look at line 93.

File: spec/features/remove_member_spec.rb [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+    it 'owner can not remove yourself' do
+      expect(page).to_not have_content('Remove')
+    end
+
+    context 'owner can remove members with admin role' do
+      let!(:member_admin) { create :member, :admin, user: other_user, organization: org }
+
+      before do
+        visit members_path
```
Comment (AlexanderUlitin) [inline spec/features/remove_member_spec.rb]: @yunixon I think user already is on members_path here. Look at line 13.

File: spec/features/remove_member_spec.rb [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+      end
+
+      it { expect(page).to_not have_content(member_admin.user_full_name) }
+    end
+
+    context 'owner can remove members with user role' do
+      let!(:member_user) { create :member, :user, user: other_user, organization: org }
+
+      before do
+        visit members_path
```
Comment (AlexanderUlitin) [inline spec/features/remove_member_spec.rb]: @yunixon I think user already is on members_path here. Look at line 13.

File: spec/features/remove_member_spec.rb [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+  let(:other_user) { create :user }
+  let!(:org)       { create :organization }
+
+  context 'when member with owner role' do
+    let!(:member_owner) { create :member, :owner, user: user, organization: org }
+
+    before do
+      sign_in member_owner.user
+      visit members_path
+    end
```
Comment (AlexanderUlitin) [inline spec/features/remove_member_spec.rb]: @yunixon why not move this `before` block up, to the main `describe` block? Looks like you use it in all cases in this spec.

Comment (yunixon) [inline spec/features/remove_member_spec.rb]: @AlexanderUlitin  I think no, sign_in with different roles:
sign_in member_owner.user
sign_in member_admin.user
sign_in member_user.user

**Pull request merged**